version.go

```diff
@@ -27,79 +26,29 @@ type runtimeInfo struct {
 	Platform  string `json:"platform,omitempty"`  // GOOS and GOARCH at build-time
 }
 
-type additions = func() (name string, value string)
+type versionAddition = func(format string) (name string, value any)
 
-func VersionCommand(id Identification, additions ...additions) *cobra.Command {
+func VersionCommand(id Identification, additions ...versionAddition) *cobra.Command {
```

Comment:
I think I see why a behavior was added instead of passing data, but it feels really awkward. What about still passing data but allowing for normalizing of the key based on the format (a detail that clio would deal with exclusively). In this way the items being added would look consistent with surrounding keys, and the user wouldn't need to pass functions and instead just a map (or similar).

Reply:
One of the reasons for functions is there is actual work that needs to be done (e.g. scanning the go module for syft, maybe other things), but doesn't need to be done if we're not printing the version information